common/src/main/java/dev/latvian/mods/kubejs/block/BlockStateModifyPlacementCallbackJS.java

```diff
@@ -23,7 +23,7 @@ public class BlockStateModifyPlacementCallbackJS extends BlockStateModifyCallbac
 	public BlockContainerJS block;
 
 	public BlockStateModifyPlacementCallbackJS(BlockPlaceContext context, Block block) {
-		super(block.defaultBlockState());
+		super(context.getLevel().getBlockState(context.getClickedPos()));
```

Comment:
Is it guaranteed that the block already exists at that position at that time?

Reply:
Ah, it is not. It is the previous block that was there

Reply:
I guess check if it is the previous block that was there, otherwise default state. 
@Spyeedy what was your intention with this suggested change? The only place this seems to be used is for custom placementStates for custom blocks, where the block you are placing is never the old block.

Reply:
Yes, my intention was to modify the already placed block's state, similar to how snow layer block works.